src/features/game/types/withdrawables.ts

```diff
@@ -209,9 +209,9 @@ export const WITHDRAWABLES: Record<InventoryItemName, WithdrawCondition> = {
   "Engine Core": false,
   Observatory: true,
   "Christmas Snow Globe": true,
-  "Cabbage Boy": true,
-  "Cabbage Girl": true,
-  "Wood Nymph Wendy": true,
+  "Cabbage Boy": false,
+  "Cabbage Girl": false,
+  "Wood Nymph Wendy": false, // (game) => !areAnyTreesChopped(game),
```

Comment:
maybe use the same withdraw condition as the "Collectible Bear" + the tree condition.

Reply:
True